style-guide/readme.md

```diff
@@ -0,0 +1,47 @@
+> **Goal of this guide:** to help you write to fit a specific context, purpose, or audience.
+
+# How to refine your writing style
+
+## Recognise what style is and what it's building blocks are
```

Comment:
Yeah, but still, the guide itself could use an intro I think. Than after that you can have chapters.